### PR TITLE
Add info on how to access ooniprobe's WUI

### PR DESCRIPTION
**Pull request by anadahz**

### CODE REVIEW

**Comment by anadahz** [comment]
This PR includes some minor format/spelling fixes.

File: README.md [hunk 1]
```diff
@@ -6,8 +6,13 @@ The image provides a ready to run
 [ooniprobe](https://github.com/TheTorProject/ooni-probe) powered Debian jessie
 distribution.
 
-User documentation and image installation instruction can be found [here]
-(https://ooni.torproject.org/install/lepidopter/)
+Upon copying the image to the SD card, plugging it in and connecting the
+Raspberry Pi to network and mains you should be able to view ooniprobe's
```
Comment (hellais) [inline README.md]: what is a "mains"?

Maybe this should be rephrased as, 

"connecting the Raspberry Pi to the network, power and waiting some minutes for it to boot up, you should be able [..]"

File: README.md [hunk 2]
```diff
@@ -6,8 +6,13 @@ The image provides a ready to run
...
+graphical interface by visiting this URL: [http://lepidopter.local]
```
Comment (hellais) [inline README.md]: s/graphical interface/graphical user interface/

File: README.md (hunk 1 above)
Comment (anadahz) [reply README.md]: With mains I meant power mains, your version sounds better.

**Pull request merged**